packages/itwinui-css/backstop/tests/listbox.html

```diff
@@ -0,0 +1,197 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>Listbox | iTwinUI</title>
+    <script type="module">
+      import "./assets/theme.js";
+      import "@itwin/itwinui-icons-elements/chevron-left";
+      import "@itwin/itwinui-icons-elements/chevron-right";
+      import "@itwin/itwinui-icons-elements/chevron-up";
+      import "@itwin/itwinui-icons-elements/chevron-down";
+      import "@itwin/itwinui-icons-elements/checkmark";
+    </script>
+    <style>
+      @import url("./assets/demo.css") layer(demo);
+      @import url('@itwin/itwinui-variables') layer(variables);
+      @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
+      section[id^='demo'] {
+        --_iui-listbox-menu-height: 200px;
+        width: auto;
+      }
+    </style>
+  </head>
+  <body class="iui-root">
+    <theme-button></theme-button>
+    <h1>Listbox</h1>
+    <hr />
+
+    <section id="demo-default">
+
+      <div class="iui-listbox-wrapper">
+        <div class="iui-input-container iui-with-message">
```

Comment:
_input_-container is meant for _inputs_. we do not expose it publicly (it's marked internal) and i dont think we should complicate this component even more by trying to force it into listbox.

Reply:
Search box has been removed